plugins/node/opentelemetry-instrumentation-aws-sdk/src/semconv.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Gen AI conventions
```

Comment:
Can you also help copy over comments for the copied `ATTR_*` attribute keys from the [semantic-conventions](https://github.com/open-telemetry/opentelemetry-js/blob/main/semantic-conventions/src/experimental_attributes.ts)?

[Example from a resource-detector package](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/resource-detector-aws-v1.11.0/detectors/node/opentelemetry-resource-detector-aws/src/semconv.ts#L17-L30)

Reply:
Thanks @trentm - feel free to push anything you need to this branch